src/emulator/env.ts

```diff
@@ -85,3 +87,30 @@ export async function getCredentialsEnvironment(
   }
   return credentialEnv;
 }
+
+export function maybeUseMonospacePortForwarding(emulatorInfos: EmulatorInfo[]): EmulatorInfo[] {
+  const portForwardingHost = process.env.MONOSPACE_PORT_FORWARDING_HOST;
```

Comment:
Can we make the Environment variable more generic? PORT_FORWARDING_HOST?

Reply:
Sure - renaming to FIREBASE_PORT_FORWARDING_HOST